schema.sql

```diff
@@ -10,3 +10,6 @@ CREATE TABLE animals (
 	neutered BOOLEAN,
 	weight_kg DECIMAL
 );
+
+ALTER TABLE animals 
+	ADD species varchar(100);
```

Comment:
Hi @OybekKayumov 
### **OPTIONAL**

- [ ] Please give varchar a value of 255 `varchar(255)` as per project requirement

Reply:
OK, done